packages/ui/src/forms/fieldSchemasToFormState/addFieldStatePromise.ts

```diff
@@ -334,7 +334,7 @@ export const addFieldStatePromise = async (args: AddFieldStatePromiseArgs): Prom
           },
           {
             promises: [],
-            rows: undefined,
+            rows: [],
```

Comment:
Instead of initializing an empty array here, we need to safely handle undefined rows on the client-side. This is to ensure that form state response sizes aren't bloated with values empty arrays. Relevant PR: #9388

Reply:
Just needs a test and then let's let it rip